bin/etoro-market-maker-keeper

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+dir="$(dirname "$0")"/..
+source $dir/_virtualenv/bin/activate || exit
+export PYTHONPATH=$PYTHONPATH:$dir:$dir/lib/pymaker:$dir/lib/pyexchange:$dir/lib/ethgasstation-client:$dir/lib/gdax-client
```

Comment:
I don't think we need `gdax-client` in PYTHONPATH here.  I know several other keepers include it unnecessarily.

Reply:
Removed `gdax-client`